app/src/main/res/layout/project_layout.xml

```diff
@@ -46,4 +53,66 @@
       app:backgroundTint="@color/button_secondary" />
   </RelativeLayout>
 
-</RelativeLayout>
+  <androidx.cardview.widget.CardView
```

Comment:
I think the initial constraints are off. If you navigate to the screen without service, you can still see the other button in the background and you see the bottom corners of the card.
<img src="https://user-images.githubusercontent.com/1289295/55012057-351cbd80-4fbd-11e9-89b8-07376105675b.png" width="330"/>

Reply:
So @Scollaco and I fixed the color of the rewards container and the visibility of the buttons. The only thing is that when the `RewardsFragment` is filled with rewards it fills out the view to the bottom of the `parent`. The `rewards_container` is constrained to the bottom of the parent for both scenarios but as you pointed out when there is no internet connection it makes it seem as if the container isn't chained to the bottom of the parent, since we don't have the rewards yet.

<img src=https://user-images.githubusercontent.com/16387538/55242930-a3a78880-5214-11e9-8a41-bd33bf494503.jpg  width=330/>

<img src=https://user-images.githubusercontent.com/16387538/55242938-a6a27900-5214-11e9-9647-9fc8f87344dd.jpg width=330/>